### ISSUE TITLE
Prototype Pollution Security Vulnerability

### ISSUE THREAD

**Issue by DarrenDooley** [newrelic/node-newrelic#1258]
Snyk is reporting a high severity security vulnerability with the latest version of the newrelic package.

**Snyk Report:**
```
  ✗ Prototype Pollution [High Severity][https://snyk.io/vuln/SNYK-JS-PROTOBUFJS-2441248] in protobufjs@6.11.2
    introduced by newrelic@8.14.1 > @grpc/grpc-js@1.5.7 > @grpc/proto-loader@0.6.9 > protobufjs@6.11.2
  This issue was fixed in versions: 6.11.3
```

Vulnerability info: https://snyk.io/vuln/SNYK-JS-PROTOBUFJS-2441248

**Comment by bizob2828** [comment]
@DarrenDooley you might need to update some snyk settings.  This has been fixed in [8.14.1](https://github.com/newrelic/node-newrelic/pull/1250).  If you look at your report the version of protobufjs where it has been fixed is 6.11.3 and that's what gets [installed](https://github.com/newrelic/node-newrelic/blob/main/package-lock.json#L22824)

**Comment by bizob2828** [comment]
I'm going to close this @DarrenDooley as we do not see an issue with the latest agent.  Feel free to reopen if you can provide a reproduction case
